.github/workflows/codemta2cff.yml

```diff
@@ -0,0 +1,26 @@
+name: CodeMeta2CFF
+run-name: Run CodeMeta2CFF after ${{github.event_name}} by ${{github.actor}}
+
+on:
+  push:
+    paths: ['codemeta.json']
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  CodeMeta2CFF:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v6
+      - name: Convert CFF
+        uses: caltechlibrary/codemeta2cff@main
+      - name: Commit CFF
+        uses: EndBug/add-and-commit@v9
+        with:
+          message: 'Add updated CITATION.cff from codemeta.json file'
+          add: 'CITATION.cff'
```

Comment:
```suggestion
      - name: Commit CFF
        run: "git commit CITATION.cff -m 'Add updated CITATION.cff from codemeta.json file'"
```

Would this work?

Reply:
Nope, you get 

```
Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
```

We could add that as a step, but that's replicating a lot of what's in the add-and-commut action. What's the opposition to using an external action?